### PR TITLE
Format code in chaincode docs

### PR DESCRIPTION
**Pull request by jougene**
Signed-off-by: Eugene Sinitsyn <sinitsyn.e@n-t.io>

#### Type of change

- Documentation update

### CODE REVIEW

**Review by sykesm — DISMISSED** [review]
Thanks for the pull request. Please remove the test.go from the commit. We don't store examples in the main fabric tree and we avoid code in the root of the tree.

**Comment by jougene** [comment]
Oh, its just test file, created to check code format.) Will remove it.

**Review by C0rWin — APPROVED** [review]

File: docs/source/chaincode4ade.rst [hunk 1]
```diff
@@ -159,102 +159,102 @@ in the ``CreateAsset`` function below.
...
-          Color:          color,
-          Size:           size,
-          Owner:          owner,
-          AppraisedValue: appraisedValue,
-        }
-        assetJSON, err := json.Marshal(asset)
-        if err != nil {
-          return err
-        }
+  func (s *SmartContract) UpdateAsset(ctx contractapi.TransactionContextInterface, id string, color string, size int, owner string, appraisedValue int) error {
```
Comment (nikhil550) [inline docs/source/chaincode4ade.rst:216]: The color formatting did not return for this function for some reason. All other functions look good.

Comment (jougene) [reply docs/source/chaincode4ade.rst:216]: I dont really understand, what you mean?

Comment (caod123) [reply docs/source/chaincode4ade.rst:216]: There's no color formatting in the `UpdateAsset` function in the built docs: https://hyperledger-fabric--1850.org.readthedocs.build/en/1850/chaincode4ade.html#initializing-the-chaincode (same for `DeleteAsset` and a couple other places). It looks like on master this was the case as well so it's probably not a huge deal but odd why it's not formatting on read-the-docs.

**Pull request merged**